docs/docs/en/guide/howto/datasource-setting.md

```diff
@@ -4,8 +4,8 @@
 
 We here use MySQL as an example to illustrate how to configure an external database:
 
-> NOTE: If you use MySQL, you need to manually download [mysql-connector-java driver][mysql] (8.0.16) and move it to the libs directory of DolphinScheduler
-> which is `api-server/libs` and `alert-server/libs` and `master-server/libs` and `worker-server/libs`.
+> NOTE: If you use MySQL, you need to manually download [mysql-connector-java driver][mysql] (8.0.16) and unzip then move it to the libs directory of DolphinScheduler
+> which is `api-server/libs` and `alert-server/libs` and `master-server/libs` and `worker-server/libs` and `tools/libs`.
```

Comment:
```suggestion
> which are `api-server/libs`, `alert-server/libs`, `master-server/libs` and `tools/libs`. Because DolphinScheduler worker no more accesses database, there is no need to copy it to `worker-server/libs` unless your task plugins need to access mysql database.
```